### PR TITLE
Feature/version view fix

### PR DESCRIPTION
**Pull request by garyluu**
Fix some tooltips, prevent highlighting a non-existent element, do dag stuff when theres a dag

### CODE REVIEW

File: src/app/workflow/dag/dag.component.ts [hunk 1]
```diff
@@ -155,6 +152,9 @@ export class DagComponent implements OnInit, AfterViewChecked, OnChanges {
         }
       }
     });
+    } else {
+      console.log('DAG is not truthy');
```
Comment (denis-yuen) [inline src/app/workflow/dag/dag.component.ts]: Log message is not very useful, make it more descriptive

Comment (garyluu) [reply src/app/workflow/dag/dag.component.ts]: Was for development purposes, I'll probably remove it altogether.

File: src/app/container/dockerfile/dockerfile.component.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ export class DockerfileComponent extends VersionSelector implements AfterViewChe
     }
   }
   ngAfterViewChecked() {
-    if (this.contentHighlighted && !this.nullContent) {
+    if (this.contentHighlighted && !this.nullContent && this.elementRef.nativeElement.querySelector('.highlight')) {
```
Comment (denis-yuen) [inline src/app/container/dockerfile/dockerfile.component.ts]: Not sure if this is a thing in typescript, but does `this.elementRef.nativeElement` need a nullcheck or anything similar?

Comment (garyluu) [reply src/app/container/dockerfile/dockerfile.component.ts]: I'm currently checking if it's truthy, it previously wasn't needed but somehow it is now.

**Pull request merged**